lib/src/main/java/com/knuddels/jtokkit/TokenEncoder.java

```diff
@@ -3,19 +3,22 @@
 import java.util.*;
 
 import static com.knuddels.jtokkit.TokenEncoderLarge.calculateTokensLarge;
+import static java.lang.Integer.MAX_VALUE;
+import static java.lang.Integer.parseInt;
 import static java.util.Collections.emptyMap;
 
-final class TokenEncoder {
-    public static final int DUMMY_RANK = Integer.MAX_VALUE;
-    public static final int MAX_RANK = Integer.MAX_VALUE - 1;
+public final class TokenEncoder {
+    public static final String VERY_LARGE_TOKENIZER_BYTE_THRESHOLD_KEY = "VERY_LARGE_TOKENIZER_BYTE_THRESHOLD";
```

Comment:
public should be fine, the users should also be able to override this from the outside if they really need it - though I wouldn't expose it through the API